src/LibraryManager.Build/Microsoft.Web.LibraryManager.Build.csproj

```diff
@@ -42,7 +42,7 @@
     </ItemGroup>
 
     <ItemGroup>
-      <ProjectReference Include="..\LibraryManager\Microsoft.Web.LibraryManager.csproj" />
+      <ProjectReference Include="..\LibraryManager\Microsoft.Web.LibraryManager.csproj" PrivateAssets="all" />
```

Comment:
This assembly shouldn't be included / is not needed by the package?

Reply:
The assembly is packaged into the .nupkg below.  The PrivateAssets here prevents the Build.nupkg to pull in the core package as a dependency, which would add it to the consuming project as well.